lms/envs/aws.py

```diff
@@ -176,6 +176,7 @@
 EDXMKTG_USER_INFO_COOKIE_NAME = ENV_TOKENS.get('EDXMKTG_USER_INFO_COOKIE_NAME', EDXMKTG_USER_INFO_COOKIE_NAME)
 
 LMS_ROOT_URL = ENV_TOKENS.get('LMS_ROOT_URL')
+ENTERPRISE_API_URL = ENV_TOKENS.get('ENTERPRISE_API_URL', LMS_ROOT_URL + '/enterprise/api/v1/')
```

Comment:
@kevinchugh @staubina Unless using `devstack_docker.yml`, LMS_ROOT_URL is still a mandatory setting, and an error would be thrown even if this didn't error out, so you may still need to set LMS_ROOT_URL: https://github.com/edx/edx-platform/blob/master/common/djangoapps/startup_configurations/validate_config.py#L31

I'll be opening a PR for this shortly.